### ISSUE TITLE
make artifacts available in maven central

### ISSUE THREAD

**Issue by francisdb** [joshdholtz/Sentry-Android#51]
We try to avoid adding non-default repositories. Would you consider syncing your repo with maven central?

https://bintray.com/docs/usermanual/uploads/uploads_syncingartifactswithmavencentral.html

**Comment by joshdholtz** [comment]
@francisdb Yes yes, I can try and do that 😊  I am currently in the middle of a move to a different city so I will try to get this early next week for ya 👌
